.github/workflows/percy.yml

```diff
@@ -0,0 +1,50 @@
+name: Percy Visual Regression Tests
+
+on:
+  workflow_dispatch:
+
+env:
+  BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+  BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+  PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - NODE_ENV: test
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+      # Needed to be able to run `npm ci`. It fails if we try to install libraries from a github repo
+      # e.g. "wpt": "git+https://github.com/Woopra/browser-tracker.git#v5.1.20",
+
+      - name: Reconfigure git to use HTTP authentication
+        run: >
+          git config --global url."https://github.com/".insteadOf
+          ssh://git@github.com/
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install Dependencies
+        run: npm ci
```

Comment:
`npm ci` is currently failing in our builds, we need to look into it to understand why. For now you can use `npm install`. This means that lines 24 to 32 can be deleted for now

Reply:
Noted. I shall remove.

Reply:
yes, it does

Reply:
Done.